parent/pom.xml

```diff
@@ -129,7 +129,7 @@
     <extension.version.os-maven-plugin>1.6.2</extension.version.os-maven-plugin>
 
     <!-- version against which backwards compatibility is checked -->
-    <backwards.compat.version>0.24.0</backwards.compat.version>
+    <backwards.compat.version>0.24.4</backwards.compat.version>
```

Comment:
RollingUpdateTest is currently disabled. We should enable them https://github.com/zeebe-io/zeebe/blob/0694fe7a0316f47009b7d1d24c9cb37d2f120223/upgrade-tests/src/test/java/io/zeebe/test/RollingUpdateTest.java#L278